docs/book/src/cronjob-tutorial/testdata/project/go.mod

```diff
@@ -1,11 +1,10 @@
 module tutorial.kubebuilder.io/project
 
-go 1.12
+go 1.13
 
 require (
 	github.com/go-logr/logr v0.1.0
 	github.com/robfig/cron v1.2.0
-	golang.org/x/crypto v0.0.0-20190308221718-c2843e01d9a2 // indirect
```

Comment:
My IDE is inserting this line automatically when I pulled this PR. Does it really need to be removed?

Reply:
It will be removed when we do a go mod tidy than it should not be committed.